DuckDuckGo/Subscription/Views/SubscriptionRestoreView.swift

```diff
@@ -190,37 +181,39 @@ struct SubscriptionRestoreView: View {
                     .foregroundColor(Color(designSystemColor: .textPrimary))
             }
             
-            VStack(alignment: .leading) {
-                if !viewModel.state.isAddingDevice {
-                    Text(UserText.subscriptionActivateEmailDescription)
-                        .daxSubheadRegular()
-                        .foregroundColor(Color(designSystemColor: .textSecondary))
-                    getCellButton(buttonText: UserText.subscriptionActivateEmailButton,
-                                  action: {
-                        DailyPixel.fireDailyAndCount(pixel: .privacyProRestorePurchaseEmailStart)
```

Comment:
My brain is in pain, too many changes XD, how confident are you that all pixels are still in place? without tests seems quite difficult to check